prime-router/src/main/kotlin/Report.kt

```diff
@@ -39,6 +39,12 @@ const val REPORT_MAX_ERRORS = 100
  * unique id and name as well as list of sources for the creation of the report.
  */
 class Report {
+    enum class NameFormat {
+        STANDARD,
+        APHL,
+        OHIO,
+    }
```

Comment:
If this continues to grow, might want to refactor into its own full-blown class, and pattern its style similar to 'Mappings' and 'JurisdictionFilters'.   While we're at it, we should rename Mapping to something less confusingly generic, like ValueMapping, FieldMapping, or ElementValueMapping or somesuch.

Reply:
I agree. We need to elevate this into something like Mappers, and we probably want to allow a file name to have multiple mapper values in it. Something like:

```
fileName:
    separator: _
    parts:
        - CDCPRIME
        - toTimeStamp('yyyyMMddHHmmss')
```